ui/src/hooks/useMicrophone.ts

```diff
@@ -193,15 +195,15 @@ export function useMicrophone() {
         
         // Find the audio transceiver (should already exist with sendrecv direction)
         const transceivers = peerConnection.getTransceivers();
-        devLog("Available transceivers:", transceivers.map(t => ({
+        devLog("Available transceivers:", transceivers.map((t: RTCRtpTransceiver) => ({
```

Comment:
This will do the `map` calculation even if debug logging is off... might want to not even make the call to `devLog` if not at debut level.

Reply:
Yeah, I completely understand your point. I will just remove the log